### PR TITLE
[BUG FIX] Update XINFO-CONSUMERS idle

### LINKED ISSUE #2055: XINFO CONSUMERS
https://redis.io/commands/xinfo-consumers/

The below statement is valid only when there is some data in the stream and the consumer was able to read that data:
- idle: the number of milliseconds that have passed since the consumer last interacted with the server

```
redis-19758.rc1.example.com:19758> XINFO CONSUMERS stream group
1) 1) "name"
   2) "consumer"
   3) "pending"
   4) (integer) 0
   5) "idle"
   6) (integer) 419717 << start
redis-19758.rc1.example.com:19758>
redis-19758.rc1.example.com:19758> XINFO CONSUMERS stream group
1) 1) "name"
   2) "consumer"
   3) "pending"
   4) (integer) 0
   5) "idle"
   6) (integer) 426985 << increasing
redis-19758.rc1.example.com:19758>
redis-19758.rc1.example.com:19758> XREADGROUP GROUP group consumer NOACK STREAMS stream >
(nil) << Nothing in the stream, but technically "the consumer interacted with the server".
redis-19758.rc1.example.com:19758> XINFO CONSUMERS stream group
1) 1) "name"
   2) "consumer"
   3) "pending"
   4) (integer) 0
   5) "idle"
   6) (integer) 442561 << increasing
redis-19758.rc1.example.com:19758> XADD stream * name Sara surname OConnor
"1658764796122-0"
redis-19758.rc1.example.com:19758> XINFO CONSUMERS stream group
1) 1) "name"
   2) "consumer"
   3) "pending"
   4) (integer) 0
   5) "idle"
   6) (integer) 454713 << increasing
redis-19758.rc1.example.com:19758> XREADGROUP GROUP group consumer NOACK STREAMS stream >
1) 1) "stream"
   2) 1) 1) "1658764796122-0"
         2) 1) "name"
            2) "Sara"
            3) "surname"
            4) "OConnor"
redis-19758.rc1.example.com:19758> XINFO CONSUMERS stream group
1) 1) "name"
   2) "consumer"
   3) "pending"
   4) (integer) 0
   5) "idle"
   6) (integer) 2351 << reset only when actual data read has happened.
redis-19758.rc1.example.com:19758>
```

The "idle" definition shall be rephrased mentioning that it resets on interaction only when data read occurs.

### PR DESCRIPTION
**Pull request by nermiller**
Fixes #2055 plus content restructure

### CODE REVIEW

**Comment by nermiller** [comment]
@itamarhaber closed and opened a new PR to get rid of previous commit artifacts

File: commands/xinfo-consumers.md [hunk 1]
```diff
@@ -1,29 +1,50 @@
...
-   2) "Bob"
-   3) pending
-   4) (integer) 1
-   5) idle
    6) (integer) 83841983
 ```
+
+## See also
+
+`XINFO` | `XINFO GROUPS` | `XINFO STREAM` | `XINFO HELP`
```
Comment (itamarhaber) [inline commands/xinfo-consumers.md:46]: 1. I'd rather add these to the frontmatter (e.g. as a `related` property) and not as text in the MD part
2. We need a definition of what we want to put in related commands - these aren't. `XINFO` isn't a real command, only a container (i.e. you can't call it by itself). `XINFO HELP` doesn't need to be listed IMO. The other two are related only in the sense of having the same container command, but that's about it.

**Review by itamarhaber — COMMENTED** [review]
I don't like how we're being opportunistic about the new layout - this creates inconsistency and confusion. I would rather have that a forklift effort.

**Comment by nermiller** [comment]
@itamarhaber ok to merge? I rolled back the structural changes.

**Pull request closed without merging**